--- origsrc/lxqt-globalkeys-0.9.0/ui/shortcut_selector.h	2015-01-07 07:25:33.000000000 -0600
+++ src/lxqt-globalkeys-0.9.0/ui/shortcut_selector.h	2015-04-28 19:36:37.492912600 -0500
@@ -40,7 +40,11 @@ namespace GlobalKeyShortcut
 class Client;
 }
 
-class ShortcutSelector : public QToolButton
+#ifndef SHARED_EXPORT
+#define SHARED_EXPORT Q_DECL_IMPORT
+#endif
+
+class SHARED_EXPORT ShortcutSelector : public QToolButton
 {
     Q_OBJECT
 public:
